easybuild/easyconfigs/p/python-igraph/python-igraph-0.11.4-foss-2023a.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'python-igraph'
+version = '0.11.4'
+
+homepage = 'https://igraph.org/python'
+description = """Python interface to the igraph high performance graph library, primarily aimed at complex network
+ research and analysis."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Clang', '16.0.6'),
+    ('libxml2', '2.11.4'),
+    ('zlib', '1.2.13'),
+    ('igraph', '0.10.10'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('pycparser', '2.21', {
+        'checksums': ['e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206'],
+    }),
+    ('cffi', '1.16.0', {
+        'checksums': ['bcb3ef43e58665bbda2fb198698fcae6776483e0c4a631aa5647806c25e02cc0'],
+    }),
+    ('texttable', '1.7.0', {
+        'checksums': ['2d2068fb55115807d3ac77a4ca68fa48803e84ebb0ee2340f858107a36522638'],
+    }),
+    ('cairocffi', '1.6.1', {
+        'checksums': ['78e6bbe47357640c453d0be929fa49cd05cce2e1286f3d2a1ca9cbda7efdb8b7'],
+    }),
+    ('igraph', version, {
+        'modulename': 'igraph',
+        'checksums': ['2437ae0157af6824e2e65a23f7a1fa4fbf0f3664333c72aeca4fc01b83e18483'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Comment:
This can be removed (it's exactly the same as what `PythonBundle` uses as default sanity check paths)